File: TextosJapones/src/main/java/org/jisho/textosJapones/model/enums/Site.java
package org.jisho.textosJapones.model.enums;

public enum Site {

	JAPANESE_TANOSHI("Japanese tanoshi"), 
	// TANGORIN("Tangorin"),
	JAPANDICT("JapanDict"), JISHO("Jisho");

	private String site;

	Site(String site) {
		this.site = site;
	}

	public String getDescricao() {
		return site;
	}

	// Necessário para que a escrita do combo seja Ativo e não ATIVO (nome do enum)
	@Override
	public String toString() {
		return this.site;
	}

}

File: ProcessaTexto/src/main/kotlin/br/com/fenix/processatexto/model/enums/Site.kt
package br.com.fenix.processatexto.model.enums

enum class Site(val descricao: String) {
    NENHUM("Nenhum"),
    TODOS("Todos"),
    JAPANESE_TANOSHI("Japanese tanoshi"),
    TANGORIN("Tangorin"),
    JAPANDICT("JapanDict"),
    JISHO("Jisho"),
    KANSHUDO("Kanshudo");

    // Necessário para que a escrita do combo seja Ativo e não ATIVO (nome do enum)
    @Override
    override fun toString(): String {
        return descricao
    }
}